sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,3 +182,13 @@ def test_connectivity(seed=36):
     assert_equal(_graph_is_connected(graph), True)
     assert_equal(_graph_is_connected(csr_matrix(graph)), True)
     assert_equal(_graph_is_connected(csc_matrix(graph)), True)
+
+
+def test_spectral_embedding_deterministic(seed=36):
+    """Test that Spectral Embedding is deterministic"""
+    random_state = np.random.RandomState(seed)
```

Comment:
hmm... I think I am not quite sure... lets see if any one else has a valid reason... IMO we should just bring it in... after any other dev replies so, feel free to do the same to others ( bringing the seed parameter inside in a separate commit )